AnkiDroid/src/main/java/com/ichi2/anki/pages/PagesActivity.kt

```diff
@@ -67,6 +71,39 @@ class PagesActivity : AnkiActivity() {
         setTitle(pageFragment.title)
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        val currentFragment = supportFragmentManager.findFragmentById(R.id.page_container)
+        if (currentFragment is Statistics) {
+            menuInflater.inflate(R.menu.statistics, menu)
+            return true
+        }
+        return true
+    }
+
+    fun showExportDialog(statsFragment: Statistics) {
+        AlertDialog.Builder(this).apply {
+            setMessage(R.string.export_stats_message)
+            setPositiveButton(R.string.dialog_ok) { _, _ ->
+                statsFragment.exportWebViewContentAsPDF()
+            }
+            setNegativeButton(R.string.dialog_cancel) { _, _ -> }
```

Comment:
I /think/ you can remove the lambda (needs testing)
```suggestion
            setNegativeButton(R.string.dialog_cancel)
```